createlinks

```diff
@@ -48,10 +48,12 @@ event_templates("nethserver-nextcloud-save", qw(
   /etc/httpd/conf.d/default-virtualhost.inc
   /etc/httpd/conf.d/nextcloud.conf
   /etc/httpd/conf.d/zz_nextcloud.conf
+  /etc/opt/rh/rh-php72/php-fpm.d/000-nextcloud.conf
 ));
 
 event_services("nethserver-nextcloud-save", qw(
   httpd restart
+  rh-php72-php-fpm restart
```

Comment:
Can we use `reload` instead? Is it implemented?

Reply:
yes